README.md

```diff
@@ -0,0 +1,6 @@
+# CocoaPods Podspecs for openwhisk-client-swift
+
+[![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
+#[![Build Status](https://travis-ci.org/apache/incubator-openwhisk-podspecs.svg?branch=master)](https://travis-ci.org/apache/incubator-openwhisk-podspecs)
```

Comment:
@bjustin-ibm my fault, I was attempting to comment it out (duh... markdown) so it was there when I activated it.  Let me fix.